src/features/account/AccountsScreen.tsx

```diff
@@ -230,19 +231,16 @@ const AccountsScreen = () => {
   }, [defaultAccountAddress, sortedAccounts])
 
   const toggleWalletsVisible = useCallback(() => {
+    triggerImpact('light')
     dispatch(appSlice.actions.toggleConnectedWallets())
     setSelectedBalance(undefined)
-  }, [dispatch])
+  }, [dispatch, triggerImpact])
 
-  const handleBalanceHistorySelected = useDebouncedCallback(
+  const handleBalanceHistorySelected = useCallback(
```

Comment:
Why did you take the debounce off?

Reply:
To make the chart more responsive. From petes feedback